source/assets/javascripts/locastyle/_table.js

```diff
@@ -0,0 +1,104 @@
+var locastyle = locastyle || {};
+
+locastyle.table = (function() {
+  'use strict';
+
+  var config = {
+    selector: {
+      module: '[data-ls-module=table]',
+      mainCheck: '.ls-main-check',
+      selected: '.ls-selected'
+    },
+    classes: {
+      selected: 'ls-selected'
```

Comment:
What the difference between **config.selector.seleted** and **classes.selected**? Is the same value, why not one variable to define this?